### PR TITLE
DS-2381 sitemap generator skip unreadable items

### PR DESCRIPTION
**Pull request by akinom**
see [DS 2381](https://jira.duraspace.org/browse/DS-2381)

added a new Console appender in log4j.properties such that all log DEBUG calls from org.dspace/app appear on the console as well as  dspace.log 

personally I find the standard command line utilities rather quiet - I like tracing to assure me that what is happening is actually what I think should be happening

### CODE REVIEW

File: dspace-api/src/main/java/org/dspace/app/sitemap/GenerateSitemaps.java [hunk 1]
```diff
@@ -65,215 +71,312 @@ public static void main(String[] args) throws Exception
...
+
         AbstractGenerator html = null;
         AbstractGenerator sitemapsOrg = null;
 
         if (makeHTMLMap)
         {
-            html = new HTMLSitemapGenerator(outputDir, htmlMapStem + "?map=",
-                    null);
+            html = new HTMLSitemapGenerator(outputDir, htmlMapStem + "?map=", null);
+            log.info("Generate HTLM map");
```
Comment (helix84) [inline dspace-api/src/main/java/org/dspace/app/sitemap/GenerateSitemaps.java]: HTML

File: dspace/config/log4j.properties [hunk 2]
```diff
@@ -99,6 +99,18 @@ log4j.appender.A3.MaxLogs=14
 log4j.appender.A3.layout=org.apache.log4j.PatternLayout
 log4j.appender.A3.layout.ConversionPattern=%d %-5p %c %x - %m%n
 
+###########################################################################
+# A4 is the name of the appender for dspace.app programs
+###########################################################################
+# These lines sets the logging level for dspace.app programs
```
Comment (helix84) [inline dspace/config/log4j.properties]: set

**Comment by mwoodiupui** [comment]
This likely has conflicts with the Service Architecture work.  It will need rebasing and repair.

**Comment by tdonohue** [comment]
This needs rebasing on the latest master, because of merge conflicts

**Comment by tdonohue** [comment]
Closing as this was replaced by #1289 (see DS-2381 ticket for more details)

**Pull request closed without merging**